### PR TITLE
added filter for checking the repo is using workflows

### PR DESCRIPTION
**Pull request by jspaaks**
**Description**

This PR adds a filter to exclude repositories that are not using any workflows yet.

**Related issues**:

- #5

**Instructions to review the pull request**

- Make a GitHub Access token here https://github.com/settings/tokens
- `cp secrets.txt.example secrets.txt`
- Update the value of GITHUB_TOKEN in secrets.txt
- install Node >= 14
- 
    
    ```
    cd $(mktemp -d --tmpdir cffbot-XXXXXX)
    git clone https://github.com/cffbots/filtering .
    git checkout <this-branch>
    source secrets.txt
    node query.mjs
    ```

### CODE REVIEW

File: query.mjs [hunk 1]
```diff
@@ -28,6 +26,17 @@ const includeHasCitationcff = async (url) => {
...
 
 
+const includeUsesWorkflows = async (url) => {
+
+    const [ owner, repo, ...unuseds ] = url.slice("https://github.com/".length).split('/');
+    const { data: { total_count: nworkflows } } = await octokit.rest.actions.listRepoWorkflows({
+        owner,
+        repo,
+    });
+    return nworkflows > nworkflows_minimum;
```
Comment (abelsiqueira) [inline query.mjs:36]: The naming "minimum" implies that's the acceptable number of workflows, so I think this condition should be `>=`.

Comment (jspaaks) [inline query.mjs:36]:
```suggestion
    return nworkflows >= nworkflows_minimum;
```

Comment (jspaaks) [reply query.mjs]: done

**Review by abelsiqueira — APPROVED** [review]
Thanks for the updates.

**Comment by jspaaks** [comment]
> Thanks for the updates.

No problem, thanks for reviewing

**Pull request merged**